### PR TITLE
Subtle cosmetics changes to task describe

### PR DESCRIPTION
**Pull request by chmouel**
Tried to make it a bit more clear to look at. To know which sections and which
steps we are looking at.

-C/--no-colour remove bullets, colors and  others ... (and that's automatic when we don't have a   proper TTY like on CI)

I understand that these things are subjective, so looking for comment before I go on adding it to other describe objects...

# Screenshots

## Pipeline

* Before:

image

* After:

image

## PipelineRun

* Before:

image

* After:

image

## Task

* Before:

image

* After:

image

## TaskRun

* Before

image

* After:

image

## PipelineResource

* Before

image

* After:

image

# Changes

# Submitter Checklist

- [👍🏼] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [🤷‍♂] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [🤷‍♂] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [👍🏼] Run the code checkers with `make check`
- [ 👍🏼] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

# Release Notes

```
Task description are now a. litle bit more friendly with colours and clearer view of what it describes 🌈
```

### CODE REVIEW

**Comment by chmouel** [comment]
The flaky is related to #599

**Comment by chmouel** [comment]
/retest

so weird, i cannot reproduce locally

**Comment by chmouel** [comment]
😓 

image

File: pkg/cmd/pipeline/describe.go [hunk 1]
```diff
@@ -30,53 +30,53 @@ import (
 	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-const describeTemplate = `Name:	{{ .PipelineName }}
+const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .PipelineName }}
```
Comment (danielhelfand) [inline pkg/cmd/pipeline/describe.go]: Not that this has to do with this pr, but it seems we don't have `Namespace` at the top of the template like other describe templates.

File: pkg/cmd/pipelinerun/describe.go [hunk 2]
```diff
@@ -30,60 +30,60 @@ import (
...
+{{decorate "bold" "Pipeline Ref"}}:	{{ $pRefName }}
 {{- end }}
 {{- if ne .PipelineRun.Spec.ServiceAccountName "" }}
-Service Account:	{{ .PipelineRun.Spec.ServiceAccountName }}
+{{decorate "bold" "Service Account"}}:	{{ .PipelineRun.Spec.ServiceAccountName }}
 {{- end }}
 
-Status
+{{decorate "underline bold" "Status\n"}}
 STARTED	DURATION	STATUS
```
Comment (danielhelfand) [inline pkg/cmd/pipelinerun/describe.go]: Is an extra space needed here before `STARTED`?

File: pkg/cmd/taskrun/describe.go [hunk 3]
```diff
@@ -29,75 +29,81 @@ import (
...
 {{- end }}
 {{- if ne .TaskRun.Spec.ServiceAccountName "" }}
-Service Account:	{{ .TaskRun.Spec.ServiceAccountName }}
+{{decorate "bold" "Service Account"}}:	{{ .TaskRun.Spec.ServiceAccountName }}
 {{- end }}
 
-Status
+{{decorate "underline bold" "Status"}}
+
 STARTED 	DURATION 	STATUS
```
Comment (danielhelfand) [inline pkg/cmd/taskrun/describe.go]: Is an extra space needed here before STARTED/with data underneath?

File: pkg/cmd/pipelinerun/describe.go (hunk 2 above)
Comment (danielhelfand) [reply pkg/cmd/pipelinerun/describe.go]: Looking at your screenshots helped clarify why. I see now it looks better this way.

File: pkg/cmd/taskrun/describe.go (hunk 3 above)
Comment (danielhelfand) [reply pkg/cmd/taskrun/describe.go]: Looking at your screenshots helped clarify why. I see now it looks better this way.

File: pkg/cmd/pipeline/describe.go (hunk 1 above)
Comment (chmouel) [reply pkg/cmd/pipeline/describe.go]: Agreed but i am too worries to send a change with all that flakeyness going on 😓

Comment (danielhelfand) [reply pkg/cmd/pipeline/describe.go]: @chmouel No worries! Let's do a follow up pr. I'll open an issue for this.

**Pull request merged**